pkg/ansible/runner/runner.go

```diff
@@ -332,17 +332,27 @@ func (r *runner) makeParameters(u *unstructured.Unstructured) map[string]interfa
 	return parameters
 }
 
+type RunResult interface {
+	Stdout() (string, error)
```

Comment:
It would be good to move the comments from `runResult.events` and `runResult.Stdout()` to here so they'll show up in the documentation.